__mocks__/RNGestureHandlerModule.js

```diff
@@ -0,0 +1,13 @@
+export default {
```

Comment:
I've also find out that you may need to mock those: 
```js
jest.mock('NativeModules', () => ({
  UIManager: {
    RCTView: () => {},
  },
  PlatformConstants: {
    forceTouchAvailable: false,
  },
}))
```

I'm not sure but it may help others :)

Reply:
Oh, nevermind. You've changed implementation of `GestureHandler`

Reply:
Certainly I do not need `PlatformConstants`. See the changes and you'll figure out why :).
Why do you think I do need `RCTView`?

Reply:
Oh, sorry. This seems to be obsolete. I've had this one in past and thought it's still necessary.